thrift/admin.thrift

```diff
@@ -217,3 +247,40 @@ struct DescribeClusterResponse {
   10: optional shared.SupportedClientVersions supportedClientVersions
   20: optional MembershipInfo membershipInfo
 }
+
+enum QueueType {
+  Replication,
+  Domain,
+}
+
+struct ReadDLQMessagesRequest{
+  10: optional QueueType queueType
+  20: optional i32 shardID
+  30: optional i64 (js.type = "Long") inclusiveEndMessageID
+  40: optional i32 maximumPageSize
+  50: optional binary nextPageToken
+}
+
+struct ReadDLQMessagesResponse{
```

Comment:
Can we also add `QueueType` in response? so that in future we can extend this with 
```
11: optional list<visibilityTask> visibilityTasks
```

Reply:
I think about this use datablob vs type + struct. I go with the later one b/c the history replication tasks are not raw events.

Reply:
ok